opm/autodiff/StandardWell_impl.hpp

```diff
@@ -1878,12 +1878,17 @@ namespace Opm
         const int well_index = index_of_well_;
         const WellControls* wc = well_controls_;
         const double* distr = well_controls_get_current_distr(wc);
+        const auto pu = phaseUsage();
 
-        std::vector<double> g = {1.0, 1.0, 0.01};
+        std::vector<double> g(np);
```

Comment:
I mean in this function. If the function scalingFactor() can be used directly without initializing the `g`, which is a re-initialization every time we use it, it will be great.

Reply:
I see, I will change it.